pkg/addon/template.go

```diff
@@ -201,6 +207,8 @@ type ClusterData struct {
 	MLA MLASettings
 	// CSIMigration indicates if the cluster needed the CSIMigration
 	CSIMigration bool
+	// KubeVirtInfraStorageClasses is a list of KubeVirt Infra-cluster storageClass
```

Comment:
please be more descriptive about this field, you could change the comment to something like this:
```
KubeVirtInfraStorageClasses is a list of storage classes from KubeVirt infra cluster that are used to 
initialization of user's cluster storage classes by the CSI driver kubevirt (hot pluggable disks)
```